core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/checkout/service/CheckoutServiceImpl.java

```diff
@@ -104,24 +95,4 @@ protected boolean hasOrderBeenCompleted(Order order) {
         return (OrderStatus.SUBMITTED.equals(order.getStatus()) || OrderStatus.CANCELLED.equals(order.getStatus()));
     }
 
-    /**
-    * Get an object to lock on for the given order id
-    * 
-    * @param orderId
-    * @return null if there was not already a lock object available. If an object was already in the map, this will return
-    * that object, which means that there is already a thread attempting to go through the checkout workflow
-    */
-    protected Object putLock(Long orderId) {
```

Comment:
@oleksiimiroshnyk @KatrukOV Should we leave these methods and deprecate them for now?  I would be cautious knowing that there is a remote possibility that a customer has the performCheckout method overridden with code that still references these methods.

Reply:
We thought about it and decided to go this way. We can deprecate in 6.1 and remove in 6.2. But I think better to let code fail now rather than rely on something that is not really needed/expected.
Let us know if we should remove in 6.1 or 6.2

Reply:
Let's deprecate in 6.1 and remove in 6.2.